pkgs/by-name/un/unblob/package.nix

```diff
@@ -0,0 +1,128 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  gitUpdater,
+  makeWrapper,
+  e2fsprogs,
+  jefferson,
+  lz4,
+  lziprecover,
+  lzop,
+  p7zip,
+  sasquatch,
+  sasquatch-v4be,
+  simg2img,
+  ubi_reader,
+  unar,
+  zstd,
+}:
+
+let
+  # These dependencies are only added to PATH
+  runtimeDeps = [
+    e2fsprogs
+    jefferson
+    lziprecover
+    lzop
+    p7zip
+    sasquatch
+    sasquatch-v4be
+    ubi_reader
+    simg2img
+    unar
+    zstd
+    lz4
+  ];
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "unblob";
+  version = "24.12.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "onekey-sec";
+    repo = "unblob";
+    tag = version;
+    hash = "sha256-lQiY5CDz4y2cOGh4gYjRPUC7jLbbvredEBO/vPQN5Eo=";
+    forceFetchGit = true;
+    fetchLFS = true;
+  };
+
+  strictDeps = true;
+
+  build-system = with python3.pkgs; [ poetry-core ];
+
+  dependencies = with python3.pkgs; [
+    arpy
+    attrs
+    click
+    cryptography
+    dissect-cstruct
+    lark
+    lief.py
+    python3.pkgs.lz4 # shadowed by pkgs.lz4
+    plotext
+    pluggy
+    pyfatfs
+    pyperscan
+    python-magic
+    rarfile
+    rich
+    structlog
+    treelib
+    unblob-native
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail jefferson "# jefferson" \
+      --replace-fail ubi-reader "# ubi-reader"
+  '';
```

Comment:
These two seem like ordinary Python dependencies (i.e. not `build-system`) — `pythonRemoveDeps` should work here (if not, add a comment explaining why)
```suggestion
  pythonRemoveDeps = [
    "jefferson"
    "ubi-reader"
  ];
```